frontend/package.json

```diff
@@ -1,11 +1,52 @@
 {
   "name": "alacrity-frontend",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "5.15.0",
+    "@testing-library/react": "11.2.7",
+    "@testing-library/user-event": "12.8.3",
+    "@types/jest": "26.0.24",
```

Comment:
should we move these @types to dev dependencies?

Reply:
ofc!!